rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/search.vm

```diff
@@ -8,19 +8,20 @@
 #header(${title} false "" )  
 <div id="rwiki_container">
   <div class="portletBody">
-	<div class="navIntraTool actionToolBar">
 	  <form action="?#" method="get" class="rwiki_searchForm">
+	    <ul class="navIntraTool actionToolBar">
 	    ##  $useHomeLink $usePrinterLink $useViewLink $viewLinkName 
 	    ##  $useEditLink $useInfoLink $useHistoryLink $useWatchLink $withNotification 
 	    ##  $homeBean $viewBean $resourceLoaderBean )
 	    ##            uhl  upl  uvl  vln    uel   uil  uhl  uwl  wn
 	  	 #commandlinks(true false false ${rlb.jsp_view}	false  false false false false	${homeBean} "" ${rlb} ) 
-	    <span class="rwiki_searchBox">
-	      ${rlb.jsp_search}:	
+	    <li class="rwiki_searchBox">
+	      <label for="searchField">${rlb.jsp_search}:</label>
```

Comment:
What element is this a label for? The input on line 22? If yes, then the input needs an id=searchField to match this label

Reply:
I have just add the id in the input text in a new commit. Thanks @ottenhoff !